File: Assets/scripts/heroControl.cs
using UnityEngine;
using System.Collections;

public class heroControl : MonoBehaviour {
    public float speed = 100;
	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
        if(Input.GetKeyDown(KeyCode.UpArrow)){
            transform.Translate (Vector3.up*Time.deltaTime*speed);
           
        }
        if (Input.GetKeyDown(KeyCode.DownArrow))
        {
            transform.Translate(Vector3.down * Time.deltaTime * speed);
           
        }
        if (Input.GetKeyDown(KeyCode.LeftArrow))
        {
            transform.Translate(Vector3.left * Time.deltaTime * speed);

        }

        if (Input.GetKeyDown(KeyCode.RightArrow))
        {
            transform.Translate(Vector3.right * Time.deltaTime * speed);

        }



       // transform.position = Camera.main.ScreenToWorldPoint(  new Vector3(Input.mousePosition.x + Input.mousePosition.y,0));

        UnityEngine.Debug.Log(Input.mousePosition.x+","+Input.mousePosition.y);
	}
}

File: Assets/scripts/Js/heroControl.js
﻿#pragma strict
var zidan:GameObject;
 var timeTemp_:float;
function Start () {

}

function Update () {
	
	timeTemp_ +=Time.deltaTime;
		
		if(Input.GetKey(KeyCode.LeftArrow)){
			transform.position.x-= 1*Time.deltaTime;
		
		}
		if(Input.GetKey(KeyCode.RightArrow)){
			transform.position.x+= 1*Time.deltaTime;
		}
		if(timeTemp_>=0.6){
		
		var n:GameObject = Instantiate(zidan,Vector3(transform.position.x,transform.position.y+0.5,0),transform.rotation);
		timeTemp_ = 0;
		}
		

}


function OnCollisionEnter2D(col:Collision2D){

}